app/__init__.py

```diff
@@ -1,7 +1,9 @@
 from flask import Flask
+from flask_cors import CORS
 from app.config import Config
 
 app = Flask(__name__)
+CORS(app)  # TODO: limit to Pyrog URIs ?
```

Comment:
For the TODO something like?
`CORS(app, resources={r"/*": {"origins": ["*.arkhn.org", "*.arkhn.com", "*.ark.hn", "127.0.0.1"]}})`

Reply:
Yes, pourquoi pas (avec localhost plutôt que 127.0.0.1 je pense ?)

Reply:
Done !

En fait il faut plutôt utiliser le paramètre `origins` que `resources`, et les regexp doivent matcher le `http(s)://` et le `:1234` s'il y a